src/cloud/live/stage-prod-test.md

```diff
@@ -206,7 +206,7 @@ Before launching, we highly recommend performing extensive traffic and performan
 
 Before you begin testing, enter a ticket with support advising the environments you are testing, what tools you are using, and the time frame. Update the ticket with results and information to track performance. When you complete testing, add your updated results and note in the ticket testing is complete with a date and time stamp.
 
-We recommend that you review the [Magento Performance Toolkit]({{ site.mage2bloburl }}/setup/performance-toolkit) options as part of your pre-launch readiness process.
+We recommend that you review the [Magento Performance Toolkit](https://github.com/magento/magento2/tree/2.3/setup/performance-toolkit) options as part of your pre-launch readiness process.
```

Comment:
use `{{ site.mage2bloburl }}/{{ site.version }}` instead of `https://github.com/magento/magento2/tree/2.3` to always point to the mainline version.